test/storage/google-object-storage.js

```diff
@@ -0,0 +1,158 @@
+      .reply(200, { access_token: 'ya29.ElqJBAAelSesAePF9jIyypEhlUZ_zuhSgWvN-8bQxNKaAmKrh6dprzp-RqKGPb-FIcCKm7zapgh_J7Ac0U75qJwCJsLtLibHHSRpmLfAZKbE1VL6GxzrxR9-CQ4', expires_in: 3600, token_type: 'Bearer' }, ['Content-Type',
+        'application/json; charset=utf-8',
+        'X-Content-Type-Options',
+        'nosniff',
+        'Cache-Control',
+        'no-cache, no-store, max-age=0, must-revalidate',
+        'Pragma',
+        'no-cache',
+        'Expires',
+        'Mon, 01 Jan 1990 00:00:00 GMT',
+        'Date',
+        'Sun, 16 Jul 2017 13:46:18 GMT',
+        'Content-Disposition',
+        'attachment; filename="json.txt"; filename*=UTF-8\'\'json.txt',
+        'Server',
+        'ESF',
+        'X-XSS-Protection',
+        '1; mode=block',
+        'X-Frame-Options',
+        'SAMEORIGIN',
+        'Alt-Svc',
+        'quic=":443"; ma=2592000; v="39,38,37,36,35"',
+        'Accept-Ranges',
+        'none',
+        'Vary',
+        'Accept-Encoding',
+        'Connection',
+        'close']);
+
+    nock('https://www.googleapis.com:443', { encodedQueryParams: true })
+      .post('/storage/v1/b', { location: 'US-CENTRAL1', name: 'ubuntu-httpx', storageClass: 'REGIONAL' })
+      .query({ project: 'network-feed' })
+      .reply(
+        200, {
+          kind: 'storage#bucket',
+          id: 'ubuntu-httpx',
+          selfLink: 'https://www.googleapis.com/storage/v1/b/ubuntu-httpx',
+          projectNumber: '98211670959',
+          name: 'ubuntu-httpx',
+          timeCreated: '2017-07-16T13:46:19.011Z',
+          updated: '2017-07-16T13:46:19.011Z',
+          metageneration: 1,
+          location: 'US-CENTRAL1',
+          storageClass: 'REGIONAL',
+          etag: 'CAE=',
+        },
+      ['X-GUploader-UploadID',
+        'AEnB2UpNYj3CiWhmXmXRaKVtXa1MU3T-6grP7PPfp_V6DFE0pXYTRvuH-7_xQe1VAa1pDVkbCpY0CFNhIRgbwm44owtxNHeKzC0WMxOn1wbfgSMANph0SLU',
+        'ETag',
+        'CAE=',
+        'Vary',
+        'Origin',
+        'Vary',
+        'X-Origin',
+        'Content-Type',
+        'application/json; charset=UTF-8',
+        'Cache-Control',
+        'no-cache, no-store, max-age=0, must-revalidate',
+        'Pragma',
+        'no-cache',
+        'Expires',
+        'Mon, 01 Jan 1990 00:00:00 GMT',
+        'Date',
+        'Sun, 16 Jul 2017 13:46:19 GMT',
+        'Content-Length',
+        '362',
+        'Server',
+        'UploadServer',
+        'Alt-Svc',
+        'quic=":443"; ma=2592000; v="39,38,37,36,35"',
+      ]);
+
+    const gcsBucketToCreate = ncGoogle.bucket();
+
+    const params = {
+      bucketName: 'ubuntu-httpx',
+      metaData: {
+        location: 'US-CENTRAL1',
+        regional: true,
+      },
+    };
+
+    gcsBucketToCreate.create(params).then((res) => {
+      console.log(res);
+      // assert.equal(res.Bucket.Id, 'ncbucketcr');
+    });
+  });
+
+  // it('should delete bucket', (done) => {
```

Comment:
Is this commented code needed?

Reply:
Examples are yet to be refactored.
Haven't changed the examples code yet.